invenio_cli/helpers/commands.py

```diff
@@ -333,6 +333,25 @@ def _lock_python_dependencies(self, pre):
         if not locked:
             subprocess.run(command, check=True)
 
+    def destroy(self):
+        """Destroys a virtualenv (if created with pipenv) and containers."""
+        try:
+            subprocess.run(['pipenv', '--rm'], check=True)
+            click.secho('Virtual environment destroyed', fg='green')
+
+        except subprocess.CalledProcessError:
+            click.secho('The virtual environment was '
+                        'not removed as it was not '
+                        'created by pipenv', fg='red')
+
+        docker_helper = DockerHelper(
+            self.cli_config.get_project_shortname(),
+            local=True)
```

Comment:
🤔 `local=True` in containerize... suspicious ;)

When you refactor this into a function, you might be tempted to think *oh this is different*. But not really... Could it just be a parameter forwarding solution? ;)